src/plone/restapi/__init__.py

```diff
@@ -1,3 +1,32 @@
 # -*- coding: utf-8 -*-
 from AccessControl import allow_module
 allow_module('json')
+
+from Products.Archetypes.ArchetypeTool import process_types, listTypes
```

Comment:
This is a hard dependency on Archetypes. I would really like us to keep Archetypes optional when people start with new projects from the scratch. I don't mind Archetypes support in plone.restapi but I don't want to force people to install legacy packages. :)